go/stellar/stellarsvc/service_test.go

```diff
@@ -598,11 +598,11 @@ func setupTestsWithSettings(t *testing.T, settings []usetting) ([]*TestContext,
 		tc2 := &TestContext{
 			TestContext: tc,
 			Fu:          fu,
+			Srv:         New(tc.G, newTestUISource()),
 			// All TCs in a test share the same backend.
 			Backend: bem,
 		}
-		rcm := NewRemoteClientMock(tc2, bem)
-		tc2.Srv = New(tc.G, newTestUISource(), rcm)
+		tc.G.GetStellar().(*stellar.Stellar).Remoter = NewRemoteClientMock(tc2, bem)
```

Comment:
This seems awkward.  Why do you need that cast in the middle?  Maybe just add SetRemoter().

Reply:
`libkb` shouldn't depend on `stellar/remote`. I suppose `Remoter interface` could be moved to `libkb` but that seems like a lot just for mocks.